frontend/src/apps/qotd/containers/Routes.js

```diff
@@ -8,21 +7,15 @@ import { selectApps } from 'core/app/reducers/orchestrationReducer'
 import Answer from './Answer'
 import Result from './Result'
 
-class Routes extends React.Component {
-  render () {
-    return this.props.render([
-      { path: '/', component: Answer },
-      { path: 'result', component: Result }
-    ])
-  }
+function Routes(props) {
+  return props.render([
+    { path: '/', component: Answer },
+    { path: 'result', component: Result },
+  ])
 }
 
-const selector = createSelector(
-  selectOrchestration,
-  selectApps,
-  apps => ({
-    apps
-  })
-)
+const selector = createSelector(selectOrchestration, selectApps, (apps) => ({
+  apps,
+}))
```

Comment:
You should not alter code that is outside the scope of the commit. I suspect the change is a result of autoformatting in your editor.

This will no longer be an issue once we implement a code formatter (i.e. Prettier).